pkg/cmd/stack/translate_test.go

```diff
@@ -346,7 +347,7 @@ func Test_translateStatefulSet(t *testing.T) {
 	if !reflect.DeepEqual(vct.Spec, volumeClaimTemplateSpec) {
 		t.Errorf("Wrong statefulset volume claim template: '%v'", vct.Spec)
 	}
-
+	assert.Equal(t, result.Spec.UpdateStrategy.Type, appsv1.RollingUpdateStatefulSetStrategyType)
```

Comment:
Is the rolling update the default for statefulsets? I don't see a change for statefulsets but we are adding this check

Reply:
Yes, we already have it as the default one for sfs

Reply:
I don't know where I checked but I don't see it know, so I will add it too

Reply:
It is important to not change the current behavior for sfs

Reply:
Checked that the default behavior is rolling update, so we still have the same behavior